### PR TITLE
Math magicians: events

### PR DESCRIPTION
**Pull request by jorgegoco**
- Copied the files calculate.js and operate.js into the logic/ directory.
- Imported the files in the Calculator component.
- Implemented the event handlers to use the math logic from calculate.js and operate.js in the React component.
- Fixed linter errors.

### CODE REVIEW

File: src/Components/Calculator.js [hunk 1]
```diff
@@ -1,34 +1,43 @@
...
-        <button className="three" type="button">3</button>
-        <button className="plus" type="button">+</button>
-        <button className="zero" type="button">0</button>
-        <button className="dot" type="button">.</button>
-        <button className="equal" type="button">=</button>
+        <div className="output">{result === '' ? '0' : result}</div>
+        <button className="ac" type="button" onClick={this.handleClick}>AC</button>
+        <button className="plus-minus" type="button" onClick={this.handleClick}>+/-</button>
+        <button className="percentage" type="button" onClick={this.handleClick}>%</button>
+        <button className="division" type="button" onClick={this.handleClick}>/</button>
```
Comment (codepantha) [inline src/Components/Calculator.js]: - [ ] I noticed a bug 🐛 when I attempted to use the `divide /` operator in your calculator app. Please look at the console error below for more information 👇 
image

File: src/Components/Calculator.js [hunk 2]
```diff
@@ -1,34 +1,43 @@
...
+        <button className="seven" type="button" onClick={this.handleClick}>7</button>
+        <button className="eight" type="button" onClick={this.handleClick}>8</button>
+        <button className="nine" type="button" onClick={this.handleClick}>9</button>
+        <button className="product" type="button" onClick={this.handleClick}>*</button>
```
Comment (codepantha) [inline src/Components/Calculator.js]: - [ ] I also noticed another bug 🐛 when I attempted to use the `multiply *` operator in your app. Please take a look at the console error below for more information 👇 
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @jorgegoco 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ No linter errors


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @jorgegoco 

Your project is complete! Great job for making the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- Calculator works as expected
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**